schemas/input/csl-citation.json

```diff
@@ -70,120 +70,114 @@
   "required": ["schema", "citationID"],
   "additionalProperties": false,
   "definitions": {
-    "locators": {
-      "title": "Locators List",
-      "description": "A list of locators. Typically, there will only be one, but there are cases where there may be more than one.",
-      "examples": [
+    "locator-variable": {
+      "description": "A locator is a list, that can be a string, or a (preferred) array.",
```

Comment:
With `locator-variable` being used to define both the whole array and the individual sub-elements, I think the "preferred" language is a little confusing. 

In the example `{ "page": [2, 3, [5, 12]] }`, `"page"` is an array with three elements, each defined to be a `locator-variable`. The elements are, in order, a number, a number, and an array with two child `locator-variable` elements, each of which is a number. Some more clarity is needed here.